libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1042,22 +1042,25 @@ string ABIFunctions::abiEncodingFunctionFunctionType(
 		return createFunction(functionName, [&]() {
 			return Whiskers(R"(
 				function <functionName>(addr, function_id, pos) {
+					addr, function_id := <convert>(addr, function_id)
 					mstore(pos, <combineExtFun>(addr, function_id))
 				}
 			)")
 			("functionName", functionName)
 			("combineExtFun", m_utils.combineExternalFunctionIdFunction())
+			("convert", m_utils.conversionFunction(_from, _to))
 			.render();
 		});
 	else
 		return createFunction(functionName, [&]() {
 			return Whiskers(R"(
 				function <functionName>(addr_and_function_id, pos) {
-					mstore(pos, <cleanExtFun>(addr_and_function_id))
+					mstore(pos, <convert>(<cleanExtFun>(addr_and_function_id)))
 				}
 			)")
 			("functionName", functionName)
 			("cleanExtFun", m_utils.cleanupFunction(_to))
```

Comment:
This already uses `_to`

Reply:
What do you mean? Is the new conversion redundant?

Reply:
No, you first clean according to the "to" type, then you convert from "from" to "to".